src/organisationUnit/organizationUnitClient.ts

```diff
@@ -65,4 +75,16 @@ export class OrganizationUnitClient extends AbstractRestfulClient {
 
     return await this.delete(parameters);
   }
+
+  public async batchUpdate(
+    action: OrganizationUnitClientActionFields,
+    organizationUnitRef: string,
+    payload: OrganizationUnitClientActionType,
+    parameters: Parameters = {},
+  ): Promise<void> {
+    this.basePath = '/organizationUnit';
```

Comment:
Why do you need to do this ?

Reply:
because organization are in multiple stack now, and basePath is not the same, in the future, all organisation unit are in the same stack, but not today

Reply:
Okay so it look like it is mandatory to reset the basePath in you method.
This way you should reset it to this value after the http request it done.
Otherwise other methods of the class with apply you custom basePath and will create 404.

Reply:
`    this.basePath = '/organizationUnit';
    this.path = `/${action}/${organizationUnitRef}`;

    const res =  await this.patch(payload, parameters);
    this.basePath = '/partners/organizationUnits';
    return res;
    `

Reply:
Maybe create a constant with instead of code double.
'/partners/organizationUnits'

Reply:
Done

Reply:
good for me